packages/ui/stories/BuiBitcoinQrDisplay.stories.js

```
@@ -175,3 +165,47 @@ export const CustomImages = {
+    selector: 'dots',
+  },
+  parameters: {
+    docs: {
+      description: {
+        story: 'Placeholder state for loading QR data. Shows skeleton UI with gray placeholder elements while data is being fetched. Use this state when loading Bitcoin addresses or Lightning invoices.'
+      }
+    }
+  }
+};
+
+export const Error = {
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)